### PR TITLE
Add linter coverage to pkg/network/multus

### PR DESCRIPTION
**Pull request by orelmisan**
### What this PR does
Before this PR:

After this PR:

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [X] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
None
```

### CODE REVIEW

File: pkg/network/multus/nad.go [hunk 1]
```diff
@@ -25,9 +25,10 @@ import (
 	"kubevirt.io/client-go/precond"
 )
 
-func GetNamespaceAndNetworkName(namespace string, fullNetworkName string) (string, string) {
+func GetNamespaceAndNetworkName(namespace, fullNetworkName string) (nadNamespace, nadName string) {
```
Comment (ormergi) [inline pkg/network/multus/nad.go:28]: Please avoid having named return values if its not a must.

**Review by ormergi — CHANGES_REQUESTED** [review]
Thanks for addressing this

File: pkg/network/multus/nad.go (hunk 1 above)
Comment (orelmisan) [reply pkg/network/multus/nad.go:28]: This change was per the linter's demand.
@EdDev do we want to relax the linter?

Comment (EdDev) [reply pkg/network/multus/nad.go:28]: No, we need to solve it by not returning two or more values of the same type.

Comment (EdDev) [reply pkg/network/multus/nad.go:28]: Either return a struct or create two types from strings.
Probably a struct is simpler.

Comment (ormergi) [reply pkg/network/multus/nad.go:28]: I would just relax the linter we know better what we want to enforce, no need to go hard on ourselves.
Returning multiple vars of the same times is not common in the codebase anyways.

Comment (orelmisan) [reply pkg/network/multus/nad.go]: The function now returns a struct instead of two strings.

**Comment by orelmisan** [comment]
[Change](https://github.com/kubevirt/kubevirt/compare/479121f73c76a5f72e315a457ee03aa7995ecda7..03563dc0adb9926be5673bef88cdd82c9acf5d85): Answered review [comment](https://github.com/kubevirt/kubevirt/pull/13038#discussion_r1793869478).

Adjusted the functions to have a uniform parameter style.

**Review by EdDev — COMMENTED** [review]
/approve

**Review by nirdothan — APPROVED** [review]

**Pull request merged**